Gds/test/fprime_gds/common/gds_cli/TestDictionary.xml

```diff
@@ -0,0 +1 @@
+<!-- Blank as a blocked writer's book -->
```

Comment:
What is this for?

Reply:
Ah, this was just for some parser testing functionality (the parser would exit early if it couldn't find an XML or .py dictionary, so I added this as a dummy dictionary for that). I'll try to remove this (or put something more descriptive inside at least if I can't delete it without breaking tests)